doc/src/manual/performance-tips.md

```diff
@@ -1208,7 +1208,7 @@ following additional properties:
   * The loop must be an innermost loop.
   * The loop body must be straight-line code. This is why `@inbounds` is currently needed for all
     array accesses. The compiler can sometimes turn short `&&`, `||`, and `?:` expressions into straight-line
-    code, if it is safe to evaluate all operands unconditionally. Consider using [`ifelse()`](@ref)
+    code, if it is safe to evaluate all operands unconditionally. Consider using [`ifelse`](@ref)
```

Comment:
Maybe worth specifying "the `ifelse` function" since people may believe it's related to `if... else` blocks.